Gulpfile.js

```diff
@@ -1,100 +1,111 @@
 
 /* eslint import/no-extraneous-dependencies: 0 */
-const babel = require('rollup-plugin-babel');
-const connect = require('connect');
-const eslint = require('gulp-eslint');
-const fs = require('fs');
-const gulp = require('gulp');
-const header = require('gulp-header');
-const http = require('http');
-const minify = require('uglify-js').minify;
-const mochaPhantomJS = require('gulp-mocha-phantomjs');
-const path = require('path');
-const rollup = require('rollup-stream');
-const serveStatic = require('serve-static');
-const source = require('vinyl-source-stream');
-const uglify = require('rollup-plugin-uglify');
+const babel = require('rollup-plugin-babel')
+const rollup = require('rollup')
+const connect = require('connect')
+const eslint = require('gulp-eslint')
+const fs = require('fs')
+const gulp = require('gulp')
+const http = require('http')
+const mochaPhantomJS = require('gulp-mocha-phantomjs')
+const path = require('path')
+const serveStatic = require('serve-static')
+const uglify = require('rollup-plugin-uglify')
 
-var parentServer; // eslint-disable-line no-var
-var childServer; // eslint-disable-line no-var
+var parentServer // eslint-disable-line no-var
+var childServer // eslint-disable-line no-var
 
-const pkg = require('./package.json');
+const pkg = require('./package.json')
 
-const banner = ['/**',
-  ' * <%= pkg.name %> - <%= pkg.description %>',
-  ' * @version v<%= pkg.version %>',
-  ' * @link <%= pkg.homepage %>',
-  ' * @author <%= pkg.author %>',
-  ' * @license <%= pkg.license %> */',
-  ''].join('\n');
+const banner = `/**
+  * ${pkg.name} - ${pkg.description}
+  * @version ${pkg.version}
+  * @link ${pkg.homepage}
+  * @author ${pkg.author}
+  * @license ${pkg.license} */
+`
+const uglifySetup = {
+  output: {
+    comments(node, comment) {
+      const text = comment.value
+      const type = comment.type
+      if (type === 'comment2') return /@preserve|@license|@cc_on/i.test(text)
+      return false
+    },
+  },
+}
 
-gulp.task('do-build', () =>
-  rollup({
-    entry: './src/postmate.js',
-    format: 'umd',
-    moduleName: 'Postmate',
+gulp.task('do-build', () => {
+  return rollup.rollup({
+    input: './src/postmate.js',
     plugins: [
       babel({
         exclude: 'node_modules/**',
       }),
-      uglify({}, minify),
+      uglify(uglifySetup),
```

Comment:
@yowainwright Are we no longer minifying things?

Reply:
Yes, we are! ...And we might be able to more now that we are using actual rollup via [babel-minify](https://github.com/babel/minify).